moped-editor/package.json

```diff
@@ -64,7 +67,7 @@
   },
   "scripts": {
     "start": "env-cmd -e local npm run start:local",
-    "start:local": "export HTTPS=true && react-app-rewired start",
+    "start:local": "react-app-rewired start",
```

Comment:
We really need to keep the HTTPS, this is because any communications between the Hasura server and the local client in HTTP will be unencrypted (plain text), and we would be passing important admin keys or JWT tokens.

I suppose it is ok when running agains the Heroku Hasura instance for testing, but let's put it back in there for now.

Reply:
I took it out to run my dev server as it kept giving me an error. I will put it back before committing next time.